android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/Interactor.kt

```diff
@@ -41,8 +41,11 @@ abstract class Interactor<P : Any, R : Router<*>> : LifecycleScopeProvider<Inter
   @Inject
   lateinit var injectedPresenter: P
   internal var actualPresenter: P? = null
-  private val lifecycleFlow = MutableSharedFlow<InteractorEvent>(1, 0, BufferOverflow.DROP_OLDEST)
-  private val lifecycleObservable = lifecycleFlow.asObservable()
+  private val _lifecycleFlow = MutableSharedFlow<InteractorEvent>(1, 0, BufferOverflow.DROP_OLDEST)
```

Comment:
qq: why using underscore for instance class variable? The IDE will easily indicate this

Reply:
Just [Kotlin coding convention for private backing properties](https://kotlinlang.org/docs/coding-conventions.html#names-for-backing-properties)

Reply:
My bad, missed this was a backing properties. Thanks for sharing